beis_indicators/utils/nuts_utils.py

```diff
@@ -0,0 +1,107 @@
+import json
+import pandas as pd
+import numpy as np
+
+from beis_indicators.utils.set_utils import set_containment
+
+NUTS_ENFORCED = {
+    2003: 2003,
+    2006: 2008,
+    2010: 2012,
+    2013: 2015,
+    2016: 2018
+}
+
+NUTS_IDS ={
+        2003: {'UKC2', 'UKK4', 'UKL2', 'UKG3', 'UKK2', 'UKD1', 'UKF2', 'UKG1', 
```

Comment:
By looking at this dict, I would rename this utilities to pointt out that they work on UK and NUTS2 only and I'd edit the title of both the issue and the PR.

Here I'd use `NUTS2_UK_IDS`.

Reply:
👍